ui/client/types/process.ts

```diff
@@ -29,4 +29,9 @@ export type ProcessDefinitionData = {
   nodesConfig?: $TodoType,
   nodesToAdd?: NodesGroup[],
   processDefinition?: $TodoType,
+  servicesDefinition?: ServicesDefinition
+}
+
+export type ServicesDefinition = {
```

Comment:
Why ServicesDefinition? Currently it's indeed for services, but in the future we may apply async also for some other stuff?

Reply:
On this level there are only "containers" for values so I was looking for some "container" that will be useful for other settings in the future... but indeed we can start with `defaultAsyncInterpretation: Boolean` and wrap it with container when we will have more then one similar field. WDYT?

Reply:
:+1: